pom.xml

```
@@ -165,6 +165,7 @@
     <module>cli</module>
     <module>gui</module>
     <module>url-updater</module>
+      <module>security</module>
```

Don't we have IDEasy configured to autoformat XML and fix indentation?
```suggestion
    <module>security</module>
```